### PR TITLE
Mobile: add button to follow/unfollow tags

### PR DESCRIPTION
**Pull request by svbergerem**
and refactor js-routes because of camelCase errors in jshint.

It looks like one needs to run `bin/rake tmp:cache:clear` to be able to use the new js-routes paths.

### CODE REVIEW

File: app/assets/javascripts/app/views/help_view.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ app.views.Help = app.views.StaticContentView.extend({
     this.CHAT_SUBS = {
       add_contact_roster_a: {
         toggle_privilege: this.getChatIcons(),
-        contacts_page: this.linkHtml(Routes.contacts_path(), Diaspora.I18n.t('chat.contacts_page'))
+        contacts_page: this.linkHtml(Routes.contacts(), Diaspora.I18n.t("chat.contacts_page"))
```
Comment (svbergerem) [inline app/assets/javascripts/app/views/help_view.js]: This is part of the translations, so I guess I won't fix it in this PR.

File: spec/javascripts/app/models/person_spec.js [hunk 2]
```diff
@@ -1,69 +1,69 @@
...
 
   context("#block", function() {
     it("POSTs a block to the server", function() {
-      this.sharing_contact.block();
+      this.sharingContact.block();
       var request = jasmine.Ajax.requests.mostRecent();
 
       expect(request.method).toEqual("POST");
-      expect($.parseJSON(request.params).block.person_id).toEqual(this.sharing_contact.id);
+      expect($.parseJSON(request.params).block.person_id).toEqual(this.sharingContact.id);
```
Comment (svbergerem) [inline spec/javascripts/app/models/person_spec.js]: This is not in camel case because it is data which is sent to the server. Again: won't fix.

**Comment by svbergerem** [comment]
This PR should be ready to review and merge.

mobile_tag_follow
mobile_tag_stop_following

File: app/assets/javascripts/app/views/help_view.js (hunk 1 above)
Comment (jhass) [inline app/assets/javascripts/app/views/help_view.js]: That it detects that in a string at all seems like a bug...

Comment (svbergerem) [inline app/assets/javascripts/app/views/help_view.js]: "contacts_page" occurs two times in that line.

**Comment by denschub** [comment]
Merged, thanks!

**Pull request closed without merging**